### PR TITLE
update bert example to compile_stage

### PR DESCRIPTION
**Pull request by eddogola**
## Description

update the bert example from pipeline driver to use compile_stage

## Checklist:
- [x] Has code been commented, particularly in hard-to-understand areas?

### CODE REVIEW

**Comment by eddogola** [comment]
see current error here: https://www.internalfb.com/phabricator/paste/view/P797734541

File: examples/hf/bert/new_pippy_bert.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+def run_worker(args):
+    torch.manual_seed(25)
+
+    bert = BertLMHeadModel(BertConfig(is_decoder=True))
+    bert.to(args.device)
+    print(bert.config)
+
+    output_chunk_spec = (TensorChunkSpec(0), sum_reducer)
+
+    split_policy = pippy.split_into_equal_size(args.world_size)
```
Comment (kwen2501) [inline examples/hf/bert/new_pippy_bert.py:25]: `split_policy` is not given to `compile_stage`

File: examples/hf/bert/new_pippy_bert.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+import argparse
+import os
+
+from transformers import BertLMHeadModel, BertConfig
+
+import torch
+import torch.distributed as dist
+
+import pippy
+from pippy.hf import PiPPyHFTracer
```
Comment (kwen2501) [inline examples/hf/bert/new_pippy_bert.py:11]: we need to use `tracer= PiPPyHFTracer()` in `compile_stage`.

File: examples/hf/bert/new_pippy_bert.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+    # print("Instantiating BERT pipeline...")
+    stage = pippy.compile_stage(
+        bert,
+        args.rank,
+        args.world_size,
+        num_chunks=args.chunks,
+        device=args.device,
+        group=None,
+        example_inputs=[],
```
Comment (kwen2501) [inline examples/hf/bert/new_pippy_bert.py:35]: Maybe we need some example inputs than empty?

**Comment by kwen2501** [comment]
Stale. Closing

**Pull request closed without merging**